src/views/mfa-verify/PassCodeForm.js

```diff
@@ -87,6 +102,13 @@ define(['okta', 'q', 'views/shared/TextBox'], function (Okta, Q, TextBox) {
         click: function () {
           form.clearErrors();
           this.disable();
+          if (factorType =='call' || factorType == 'sms') {
```

Comment:
let's use `===` for comparison
and nit, space after `=` and before `'`

Reply:
fixed